homeassistant/brands/yale_us_ca.json

```
@@ -0,0 +1,5 @@
+{
+  "domain": "yale_us_ca",
+  "name": "Yale (US/Canada)",
```

```suggestion
  "name": "Yale August (US/Canada)",
```

We could call it `Yale August` since thats what the company/division is called